acra/guides/trying-acra-with-docker/_index.md

```diff
@@ -0,0 +1,403 @@
+* `acra-keymaker.dockerfile` – resulting image with AcraKeymaker tool
+* `acra-webconfig.dockerfile` – resulting image with AcraWebconfig component
+* `acra-build.dockerfile` – intermediate image for compiling all the Acra components
+* `mysql-nossl.dockerfile` – MySQL server container with disabled SSL
+* `mysql-ssl.dockerfile` – MySQL server container with example SSL certificates (located at ssl/mysql directory)
+* `postgresql-ssl.dockerfile` – Postgresql server container with example SSL certificates (located at ssl/postgresql directory)
+
+All the images are already built and uploaded to the [Docker Hub Cossack Labs repository](https://hub.docker.com/u/cossacklabs/), so you can use them in a traditional manner:
+
+```bash
+docker run <options> cossacklabs/<component> <arguments>
+```    
+
+You do not need to do it, but you may want to build all the images from current sources manually. To do that, type:
+
+```bash
+make docker
+```    
+
+After that step, you'll get the described above pre-built images (intermediate `acra-build` image will be wiped out after the build is complete).
+
+All docker images have tags. When we build them, we set appropriate tags for each image:
+
+* sliding tags:
+    - `stable` and `latest` - stable branch, recommended, default;
+    - `master` and `current` - master branch of GitHub repository.
+
+* fixed tags:
+    - `<full_commit_tag>` - specify the exact commit in the repository;
+    - `<version>` - choose version tag.
+
+An image usually has two (commit, version) or four (+ branch, latest/current) tags.
+
+#### Compose files
+
+We wanted you to be able to easily try the most useful schemes that we prepared as Docker Compose files in the `docker` subdirectory. The name of each Docker Compose file describes its components and their interconnections in a simple form. For example: `docker-compose.pgsql-nossl-server-ssession-proxy.yml` is a scheme with Postgresql DB, AcraServer, and AcraConnector, connected to AcraServer through the Secure Session link.
+
+The examples contain references to `acra-keymaker` and `acra-authmanager` containers inside. They are used for creation and distribution of the necessary keys. They were included for simplification of the test launch and should not be used in production schemes (where the keys should be generated manually and deployed to an appropriate host according to the security rules of your infrastructure).
+
+Check out the [docker folder](https://github.com/cossacklabs/acra/blob/master/docker) for examples of compose files.
+
+Most likely you would like to set some variables before launch.
+
+Please set `ACRA_MASTER_KEY` environmental variable, or check [KeyManagement]({{< ref "acra/acra-in-depth/cryptography-and-key-management/#-INVALID" >}}) page for more details:
+```bash
+
+export ACRA_SERVER_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
+export ACRA_CONNECTOR_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
+```
```

Comment:
“Master keys for AcraServer and AcraConnector can be configured separately. Please read the key management guide for more details.”

There are two master keys, these are different from `ACRA_MASTER_KEY` with non-Compose setup, and you cannot use `ACRA_MASTER_KEY` with it. You must set these variables if you want to use different keys.

Also, the empty line after ` ```bash ` should probably come before that line.